ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3048,6 +3048,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-upi-zones
+- as: vsphere-ipi-ovn-winc-f7
+  cron: 3 18 16,30 * *
```

Comment:
Frequency `fN` (every 7 days) mismatch with the cron entry ( every 14 days).
You can update the frequency `fN`, then use the [shell](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh) script to generate the cron entry, or the [python](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/update-cron-entries.py) one to update them in batch.

Reply:
Got you, updated to f14. Thanks to the tools advice